create_jenkins_job.py

```diff
@@ -296,6 +296,112 @@ def create_job(os_name, job_name, template_file, additional_dict):
             })
 
         # configure a manually triggered version of the coverage job
+
+        # Proposed list of packages to maximize coverage while testing quality level
+        # packages. The list is composed by the list of qualitly level packages plus
+        # packages of ros2.repos that are used by the qualitly level packages during
+        # tests.
+
+        # out of the list since ignored by colcon: shape_msgs, stereo_msgs, rmw_connext, rmw_cyclonedds
+        quality_level_pkgs = ['action_msgs',
+                              'ament_index_cpp',
+                              'builtin_interfaces',
+                              'class_loader',
+                              'composition_interfaces',
+                              'console_bridge_vendor',
+                              'diagnostic_msgs',
+                              'foonathan_memory_vendor',
+                              'geometry_msgs',
+                              'libstatistics_collector',
+                              'libyaml_vendor',
+                              'lifecycle_msgs',
+                              'nav_msgs',
+                              'rcl',
+                              'rcl_action',
+                              'rcl_interfaces',
+                              'rcl_lifecycle',
+                              'rcl_logging_spdlog',
+                              'rcl_yaml_param_parser',
+                              'rclcpp',
+                              'rclcpp_action',
+                              'rclcpp_components',
+                              'rclcpp_lifecycle',
+                              'rcpputils',
+                              'rcutils',
+                              'rmw',
+                              'rmw_dds_common',
+                              'rmw_fastrtps_cpp',
+                              'rmw_fastrtps_shared_cpp',
+                              'rmw_implementation',
+                              'rosgraph_msgs',
+                              'rosidl_default_runtime',
+                              'rosidl_runtime_c',
+                              'rosidl_runtime_cpp',
+                              'rosidl_typesupport_c',
+                              'rosidl_typesupport_cpp',
+                              'rosidl_typesupport_fastrtps_c',
+                              'rosidl_typesupport_fastrtps_cpp',
+                              'rosidl_typesupport_interface',
+                              'spdlog_vendor',
+                              'statistics_msgs',
+                              'std_msgs',
+                              'std_srvs',
+                              'stereo_msgs',
+                              'trajectory_msgs',
+                              'unique_identifier_msgs',
+                              'visualization_msgs',
+                              'fastcdr',
+                              'fastrtps',
+                              'tracetools']
```

Comment:
Why are the last few rows not in alphabetical order?

Reply:
I copy them directly from our working document where they were at the end for probably a good reason. Reorder in d9a9997.